packages/atomic-layout/test/ssr.spec.tsx

```diff
@@ -25,4 +26,13 @@ describe('Server-side rendering', () => {
 
     expect(renderOnServer).not.toThrow()
   })
+
+  it('renders with the src attribute', () => {
+    const StyledImage = styled.img`
+      border-radius: 5px;
+    `
+    const ElementImage = useResponsiveComponent(StyledImage)
```

Comment:
It's also sufficient to render `img` directly:

```jsx
const Image = useResponsiveComponent((props) => <img {...props} />)
const html = renderToString(<Image src="avatar.jpg" />)
```

I'm not against using styled components, but we should try to write less styling-specific code in the core library. If anything, we shouldn't write any style-specific code there.

Reply:
make sense, will do that